scss/4-elements/_lists.scss

```diff
@@ -10,42 +9,80 @@ li,
   }
 }
 
-ol,
-.cads-list-ordered {
+%cads-list-ordered {
   padding: 0 0 0 $cads-spacing-5;
   margin: 0 0 $cads-spacing-4 0;
+
+  li {
+    @extend %cads-list-item;
+  }
 }
 
-ul,
-.cads-list-unordered {
+%cads-list-unordered {
   padding: 0 0 0 $cads-spacing-4;
   margin: 0 0 $cads-spacing-4 0;
   list-style-type: disc;
 
-  &.cads-list-unordered__inline {
-    padding: 0;
-    margin: 0 0 $cads-spacing-4 0;
-    list-style-type: none;
+  li {
+    @extend %cads-list-item;
+  }
+}
+
+%cads-list-unordered-inline {
+  padding: 0;
+  margin: 0 0 $cads-spacing-4 0;
+  list-style-type: none;
+
+  li {
+    display: inline-block;
+    margin-right: 1rem;
+  }
+
+  // Reverse margins if this is right aligned
+  .cads-align-right & {
+    li {
+      margin-right: 0;
+      margin-left: 1rem;
+    }
+  }
+}
+
+%cads-list-no-bullet {
+  padding: 0;
+  margin: 0 0 $cads-spacing-4 0;
+  list-style-type: none;
 
-    li,
-    .cads-list-item {
-      display: inline-block;
-      margin-right: 1rem;
+  li {
+    @extend %cads-list-item;
+  }
+}
+
+.cads-list-no-bullet {
+  @extend %cads-list-no-bullet;
+}
+
+.cads-prose {
```

Comment:
Style preference: Does it make sense to have a single file for the prose styles, or happy defining this inline like this?

Reply:
yup good idea, I've made a new objects file `_prose` in section 5 👍